primer/brand
# Fix broken ComparisonTable columns on screen readers

## Summary

Removed the `aria-hidden` attribute from empty `<th>` cells which was causing table structure to be misrepresented by screen readers

## List of notable changes:

- Removed `aria-hidden` from `ComparisonTable` column headers

## What should reviewers focus on?

- Ensure the table structure is accurately represented when using a screen reader

## Steps to test:

1. Open [a ComparisonTable example on Storybook](https://primer-52bb15c8a0-26139705.drafts.github.io/brand/storybook/?path=/story/components-comparisontable--minimal).
1. Enable a screen reader (eg VoiceOver).
1. Navigate through the table and ensure that column numbers and titles are accurately communicated.

## Supporting resources (related issues, external links, etc):

- https://github.com/github/primer/issues/3753

## Contributor checklist:


## Reviewer checklist:


## Videos

Note the mis-reported column numbers in the first video.

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">


https://github.com/user-attachments/assets/90476b0d-0aa9-403c-bb84-1de472be2084



 </td>
<td valign="top">


https://github.com/user-attachments/assets/8e4d783c-4b10-45eb-a1b5-49eca2f2c575



</td>
</tr>
</table>
